portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,75 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+    private List<String> quotes;
+
+   public void init() {
+    quotes = new ArrayList<>();
+    quotes.add("Here is one message!");
+    quotes.add("Here is a second message!");
+    quotes.add("Here is a third message!");
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    String json = convertToJson(quotes);
+
+    // Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+    //response.getWriter().println("<h1>Hello Emily!</h1>");
```

Comment:
You should remove unused code instead of commenting them out and leaving them there!